plugins/obs-webrtc/whip-output.cpp

```diff
@@ -603,6 +603,12 @@ void WHIPOutput::Send(void *data, uintptr_t size, uint64_t duration,
 void register_whip_output()
 {
 	const uint32_t base_flags = OBS_OUTPUT_ENCODED | OBS_OUTPUT_SERVICE;
+#ifdef ENABLE_HEVC
+	const char *video_codecs = "h264;hevc;av1";
+#else
+	const char *video_codecs = "h264;av1";
+#endif
```

Comment:
See other comment about ENABLE_HEVC enum guards.

Reply:
Discussed off-thread. Specifically in this case for register_whip_output, this change is fine and matches other places in the application.
https://github.com/obsproject/obs-studio/blob/fad6f43608cbfb22e132b83f53ad04e45522a144/plugins/obs-ffmpeg/obs-ffmpeg-hls-mux.c#L337-L341
https://github.com/obsproject/obs-studio/blob/fad6f43608cbfb22e132b83f53ad04e45522a144/plugins/obs-outputs/rtmp-stream.c#L1894
https://github.com/obsproject/obs-studio/blob/fad6f43608cbfb22e132b83f53ad04e45522a144/plugins/obs-ffmpeg/obs-ffmpeg-mpegts.c#L1333